CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/ConfigurationPane.java

```diff
@@ -180,6 +186,20 @@ public void setSampleData(final String[] columnLabels, final List<String[]> curr
         });
     }
 
+    /**
+     * Clears the run pane that is currently selected
+     */
+    protected void clearSelectedPane() {
+        if (tabPane.getSelectionModel().getSelectedItem() != null) {
+            RunPane currentSelected = (RunPane) tabPane.getSelectionModel().getSelectedItem().getContent();
+
+            String[] columns = {};
+            List<String[]> data = new ArrayList<>();
+
+            currentSelected.setSampleData(columns, createTableRows(data));
```

Comment:
looking at the implementation of `createTableRows`, it just returns an initial observable array list if the input list is empty (as it won't do anything in the for loop). In that case, could you just pass `FXCollections.observableArrayList()` as the 2nd parameter instead? Consequently could remove the `data` list